@zooniverse/react-components: Module not found error

## Package
_Choose from the list:_
- [ ] app-content-pages
- [ ] app-project
- [ ] lib-classifier
- [ ] lib-panoptes-js
- [x] lib-react-components
- [ ] unknown

## Describe the bug
Version 1.9.0 of `@zooniverse/react-components` errors on install.

```sh
ERROR in ./node_modules/@zooniverse/react-components/dist/esm/hooks/usePanoptesUser.js 5:0-25
Module not found: Error: Can't resolve 'swr' in '/home/runner/work/community-catalog/community-catalog/node_modules/@zooniverse/react-components/dist/esm/hooks'
```

## To Reproduce
See this [broken build](https://github.com/zooniverse/community-catalog/actions/runs/6892539078/job/18750062368?pr=213) for Community Catalogue.


## Expected behavior
It should install without errors.

--- a/packages/lib-react-components/CHANGELOG.md
+++ b/packages/lib-react-components/CHANGELOG.md
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.9.1] 2023-11-16
+
+### Fixed
+Added missing `swr` package (used by Panoptes data-fetching hooks.)
+
 ## [1.9.0] 2023-11-08
 
 ### Added

--- a/packages/lib-react-components/package.json
+++ b/packages/lib-react-components/package.json
@@ -3,7 +3,7 @@
   "description": "Zooniverse React Components",
   "license": "Apache-2.0",
   "author": "Zooniverse <contact@zooniverse.org> (https://www.zooniverse.org/)",
-  "version": "1.9.0",
+  "version": "1.9.1",
   "main": "dist/cjs/index.js",
   "module": "dist/esm/index.js",
   "exports": {
@@ -53,7 +53,8 @@
     "prop-types": "~15.8.1",
     "react-i18next": "~13.5.0",
     "react-resize-detector": "~9.1.0",
-    "react-rnd": "10.4.1"
+    "react-rnd": "10.4.1",
+    "swr": "~2.2.4"
   },
   "peerDependencies": {
     "@zooniverse/grommet-theme": "3.x.x",

